cli/packages/cmd/root.go

```diff
@@ -43,14 +44,33 @@ func init() {
 	rootCmd.PersistentFlags().Bool("silent", false, "Disable output of tip/info messages. Useful when running in scripts or CI/CD pipelines.")
 	rootCmd.PersistentPreRun = func(cmd *cobra.Command, args []string) {
 		silent, err := cmd.Flags().GetBool("silent")
-		config.INFISICAL_URL = util.AppendAPIEndpoint(config.INFISICAL_URL)
 		if err != nil {
 			util.HandleError(err)
 		}
 
+		config.INFISICAL_URL = util.AppendAPIEndpoint(config.INFISICAL_URL)
+
 		if !util.IsRunningInDocker() && !silent {
 			util.CheckForUpdate()
 		}
+
+		loggedInDetails, err := util.GetCurrentLoggedInUserDetails()
+
+		if !silent && err == nil && loggedInDetails.IsUserLoggedIn && !loggedInDetails.LoginExpired {
+			token, err := util.GetInfisicalToken(cmd)
+
+			if err == nil && token != nil {
+				var usingFrom string
+				if token.PassedAsFlag {
+					usingFrom = "--token flag"
+				} else {
+					usingFrom = "INFISICAL_TOKEN environment variable"
```

Comment:
This says `INFISICAL_TOKEN` but in getInfisicalToken we also check for `INFISICAL_UNIVERSAL_AUTH_ACCESS_TOKEN_NAME`. It's better you use replace `PassedAsFlag` to something like `source`